src/pages/Hojung/Login/Login.scss

```
@@ -0,0 +1,78 @@
+* {
+  color: #333;
+  font-size: 14px;
+}
+
+.contain {
+  max-width: 975px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+.input {
+  border: 1px solid #ddd;
+  padding: 4px 8px;
+}
+
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#login {
+  @include flex-center;
+  height: calc(100vh - 55px);
+  background: #fafafa;
+  .loginWrap {
+    width: 400px;
+    padding: 45px;
+    text-align: center;
+    border: 1px solid #ddd;
+    background: #fff;
+    .sitelogo {
+      font-family: 'Lobster', cursive;
```

```suggestion
.login {
  @include flex-center;
  height: calc(100vh - 55px);
  background: #fafafa;
  
  .loginWrap {
    width: 400px;
    padding: 45px;
    text-align: center;
    border: 1px solid #ddd;
    background: #fff;
    
    .sitelogo {
      font-family: 'Lobster', cursive;
```
- 구분되는 요소들끼리 한줄씩 띄워서 구분해주면 좋을 거 같아요
- font 저 부분만 달라져야 하는 건가요?

---

네! 로고 부분만 폰트가 달라져야 합니다!